fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/RebuildIT.java

```diff
@@ -129,6 +142,59 @@ public void testRebuildOcfl() {
         assertNotContains("junk");
     }
 
+    @Test
+    public void testRebuildOnStart() throws Exception {
+        assertFalse("rebuild on start is disabled", fedoraPropsConfig.isRebuildOnStart());
+        rebuild("test-rebuild-ocfl/objects");
+
+        // Optional debugging
+        if (LOGGER.isDebugEnabled()) {
+            ocflRepository.listObjectIds().forEach(id -> LOGGER.debug("Object id: {}", id));
+        }
+
+        assertTrue("Should contain object with id: " + FedoraTypes.FEDORA_ID_PREFIX,
+                ocflRepository.containsObject(FedoraTypes.FEDORA_ID_PREFIX));
+        assertContains("binary");
+        assertContains("test");
+
+        final var binaryId = FedoraTypes.FEDORA_ID_PREFIX + "/binary";
+        final var objectVersionFile = this.ocflRepository.getObject(ObjectVersionId.head(binaryId)).getFile("binary");
+        final var path = objectVersionFile.getStorageRelativePath();
+        //delete ocfl object from disk:  path is "<object root dir>/v1/content/binary" therefore resolve the great
+        //grandparent directory
+        final var ocflObjectDirectory =
+                Path.of(TARGET_FCREPO_HOME_DATA_OCFL_ROOT, path)
+                        .toFile().getParentFile().getParentFile().getParentFile();
+        FileUtils.deleteDirectory(ocflObjectDirectory);
```

Comment:
Why not just do `ocflRepo.purgeObject(binaryId)`?

Reply:
That's a good point.  I'll do that.

Reply:
@pwinckles: this should be ready to go.  Also updated the documentation to reflect these changes.